tests/base/test_os.lua

```diff
@@ -274,6 +290,24 @@
 	end
 
 --
+-- os.translateCommand() windows LINKDIR tests
```

Comment:
```suggestion
-- os.translateCommand() LINKDIR/LINKFILE tests
```
as both windows and posix are tested, and both LINKDIR/LINKFILE.